inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/utils/SharedPreferencesUtil.kt

```diff
@@ -2,11 +2,14 @@ package com.rakuten.tech.mobile.inappmessaging.runtime.utils
 
 import android.content.Context
 import android.content.SharedPreferences
+import com.rakuten.tech.mobile.inappmessaging.runtime.data.repositories.AccountRepository
 
 internal object SharedPreferencesUtil {
 
     fun createSharedPreference(context: Context, account: String): SharedPreferences =
             context.getSharedPreferences(PREFS + "_$account", Context.MODE_PRIVATE)
 
+    fun getPreferencesFile() = PREFS + "_" + AccountRepository.instance().userInfoHash
```

Comment:
Either move this to `InApp` so you can remove `SharedPreferencesUtil`, or move all sharedpreferences related functions here.

Something like this
```
fun getString(key: String, default: String) = {
    InAppMessaging.instance().getHostAppContext()?.let { it ->
        PreferencesUtil.getString(
        it,
        getPreferencesFile(),
        key,
        default
    )} ?: default
}
```

Reply:
Updated